recipes/ecmwf-api-client/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: ecmwf-api-client
+  version: "1.4"
+
+source:
+  fn: ecmwf-api-client-python.tgz
+  url: https://software.ecmwf.int/wiki/download/attachments/56664858/ecmwf-api-client-python.tgz
```

Comment:
I don't understand the `ecmwf` software stack, so I am curious how does this package relates to: 

- https://github.com/conda-forge/ecmwf_grib-feedstock
- https://github.com/conda-forge/python-ecmwf_grib-feedstock
- https://github.com/conda-forge/eccodes-feedstock

Also, would you be interested in help maintaining any of the above?